examples/MvcWeb/TheraLang/Controllers/ResourceController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using MvcWeb.TheraLang.Entities;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.TheraLang.Services;
+
+namespace MvcWeb.TheraLang.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ResourceController : ControllerBase
+    {
+        public ResourceController(IResourceService _service)
+        {
+            this.service = _service;
+        }
+
+        private readonly IResourceService service;
+
+        [HttpPost]
+        public async Task<IActionResult> PostResource(Resource resource)
+        {
+            if(!ModelState.IsValid)
+            {
+                return BadRequest();
+            }
```

Comment:
please remove these checks from ApiControllers as it is done automatically from some .net core version **in case if [ApiController] attribute applied**.
see details:
https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-3.0